src/legend.js

```diff
@@ -160,12 +160,16 @@ export const Legend = Class.create( {
      * @param {String} name The human-readable object name or description
      * @return {HTMLLIElement} List element to be insert in the legend
      */
-  _generateElement: function(id, name) {
+  _generateElement: function(id, name, isObsolete) {
     var color = this.getObjectColor(id);
-    var item = new Element('li', {'class' : 'disorder '+'drop-'+this._getPrefix(), 'id' : this._getPrefix() + '-' + id}).update(new Element('span', {'class' : 'disorder-name'}).update(name));
+    var obsoleteClass = isObsolete ? 'obsolete ' : '';
```

Comment:
Small change - on first read I thought you'd missed a space below, before `'drop-'...`, then realised it's been added here. Might be better to leave the space out of here and add below?

Reply:
I've moved the space for better clarity.